corehq/apps/reports/standard/project_health.py

```diff
@@ -327,6 +329,49 @@ def previous_six_months(self):
             last_month_summary = this_month_summary
         return six_month_summary
 
+    def export_summary(self, six_months):
+        table = []
+        table.append('Six Month Performance Summary')
+        worksheet_headers = ['month', 'num_high_performing_users', 'num_low_performing_users',
+                             'total_active', 'total_inactive', 'total_num_users']
```

Comment:
Yup, it does. The format it follows is this: [ ['table_or_sheet_name', [['header'] ,['row']] ] ]